<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeaCodec</class>
 <widget class="QWidget" name="TeaCodec">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TeaCodec</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="layoutLeft">
     <item>
      <widget class="QGroupBox" name="groupBoxFiles">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Files</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="labelSource">
          <property name="text">
           <string>Source</string>
          </property>
          <property name="buddy">
           <cstring>editSource</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="layoutSource">
          <item>
           <widget class="QLineEdit" name="editSource">
            <property name="placeholderText">
             <string>Path to encrypted PDF file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonBrowseSource">
            <property name="toolTip">
             <string>Browse</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icons/button/file_icon.png</normaloff>:/icons/button/file_icon.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelDestination">
          <property name="text">
           <string>Destination</string>
          </property>
          <property name="buddy">
           <cstring>editDestination</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="layoutDestination">
          <item>
           <widget class="QLineEdit" name="editDestination">
            <property name="placeholderText">
             <string>Output directory for decrypted PDF file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonBrowseDestination">
            <property name="toolTip">
             <string>Browse</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icons/button/dir_icon.png</normaloff>:/icons/button/dir_icon.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="actionLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="buttonAction">
         <property name="toolTip">
          <string>Start/cancel decryption</string>
         </property>
         <property name="text">
          <string>Decrypt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="statusDump">
       <property name="font">
        <font>
         <family>Noto Sans Mono</family>
        </font>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutBruteForce">
     <item>
      <widget class="QGroupBox" name="layoutRight">
       <property name="title">
        <string>Brute force options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBoxAlphabet">
          <property name="title">
           <string>Alphabet</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="checkBoxLowercase">
             <property name="text">
              <string>Lowercase letters</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxUppercase">
             <property name="text">
              <string>Uppercase letters</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxNumbers">
             <property name="text">
              <string>Numbers</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxSpecial">
             <property name="text">
              <string>Special characters</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxSpace">
             <property name="text">
              <string>Space</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="lineSplitCustomAlphabet">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="layoutCustomAlphabet">
             <item row="0" column="0">
              <widget class="QLabel" name="labelCustomAlphabet">
               <property name="text">
                <string>Custom</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="editCustomAlphabet">
               <property name="placeholderText">
                <string>User defined alphabet</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxOrder">
          <property name="title">
           <string>Order</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioInOrder">
             <property name="text">
              <string>In order</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioShuffle">
             <property name="text">
              <string>Shuffle</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="layoutKnownKey">
          <item row="0" column="0">
           <widget class="QLabel" name="labelKnownKey">
            <property name="text">
             <string>Known key part</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="editKnownKey">
            <property name="maxLength">
             <number>16</number>
            </property>
            <property name="placeholderText">
             <string>Known beginning of the password</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxStatus">
       <property name="title">
        <string>Status dump options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="layoutStatusLevel">
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelStatusLevel">
            <property name="text">
             <string>Level</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxStatusLevel">
            <property name="currentText">
             <string>Debug</string>
            </property>
            <item>
             <property name="text">
              <string>Debug</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Info</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Warning</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="buttonClearStatus">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Clear status</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editSource</tabstop>
  <tabstop>buttonBrowseSource</tabstop>
  <tabstop>editDestination</tabstop>
  <tabstop>buttonBrowseDestination</tabstop>
  <tabstop>buttonAction</tabstop>
  <tabstop>statusDump</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
